account/models.py

```diff
@@ -1,8 +1,32 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django.utils import timezone
 
 # Create your models here.
 
 
 # class User(AbstractUser):
 #     guest = models.BooleanField(default=False)
+
+from django.contrib.auth.models import User
+
+class ProfileModel(models.Model):
+    GENDER_CHOICES=(
+       ('M',"Male"),
+       ('F',"Female"),
+       ('O',"Other"),
+       ('P',"Prefer not to say"),
+    )
+
+    user=models.OneToOneField(User,on_delete=models.CASCADE,related_name="Profile")
+    coins=models.PositiveIntegerField(default=0)
+    gender=models.CharField(max_length=1,choices=GENDER_CHOICES)
+    DOB=models.DateField(default=timezone.now)
+    TotalPlayCount=models.PositiveIntegerField(default=0)
```

Comment:
Don't capitalize fields name. name it as ``dob``, ``total_play_count``, etc.

Reply:
Here, don't set the default value of ``dob``. keep it ``blank=True`` and ``null=True``

Reply:
fine